src/app/services/files.js

```diff
@@ -220,6 +220,7 @@ module.exports = (Model, App) => {
         name: { [Op.eq]: destinationName },
         folder_id: { [Op.eq]: destination },
         type: { [Op.eq]: file.type },
+        fileId: { [Op.ne]: fileId }
```

Comment:
reason is: if recover file(move file) in same directory without this, exist throw exception because file exist, but with deleted: 1, with this we can "move" file from { destination: 4, deleted: 1 } to { destination: 4, deleted: 0}

Reply:
What would happen if that same file with the same name is recovered then?

Reply:
only updates database with same destination and deleted: 0, now every movement of files and folders set deleted: 0, with this we prevent mistakes or others, we Cant move files inside trash folders, only move files/folders from trash to outsite.